gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/TraversalHelper.java

```diff
@@ -57,6 +60,23 @@
     private TraversalHelper() {
     }
 
+    public static boolean isNotBeyondElementId(final Traversal.Admin<?, ?> traversal) {
```

Comment:
I think some javadoc here in `TraversalHelper` would be smart.  These methods are very useful in strategy development which is externally facing to providers/users.  `TraversalHelper` also has a good body of unit tests around it at this point - adding some coverage for this new method would be good too:

https://github.com/apache/incubator-tinkerpop/blob/master/gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/util/TraversalHelperTest.java